docs/onboarding/01-project-overview.md

```diff
@@ -0,0 +1,92 @@
+# Yellowstone Vixen - Project Overview
+
+## Introduction
+Yellowstone Vixen is a high-performance Rust-based project that provides a gRPC service for parsing Solana transactions. It's designed to work seamlessly with the Yellowstone ecosystem, offering efficient and reliable transaction parsing capabilities.
+
+## Architecture Overview
+
+```mermaid
+graph TB
+    Client[Client Applications] -->|gRPC Requests| Vixen[Yellowstone Vixen Service]
+    Vixen -->|Parse| Transactions[Solana Transactions]
+    Vixen -->|Metrics| Prometheus[Prometheus Monitoring]
+    Vixen -->|Tracing| OpenTelemetry[OpenTelemetry]
+    
+    subgraph "Vixen Service Components"
+        Parser[Transaction Parser]
+        ProtoGen[Protocol Buffer Definitions]
+        Metrics[Metrics Collection]
+        Config[Configuration Management]
+    end
+
+    Vixen --> Parser
+    Vixen --> ProtoGen
+    Vixen --> Metrics
+    Vixen --> Config
+```
```

Comment:
This is a bit off. What are you looking to document the flow of the software internally or pieces user should be aware of? 

I think we separate the personas and have docs tailored to each where we build out docs first for the end consumer.

Reply:
Oh I was creating this diagrams to understand the Architecture of this project while learning more about Solana ecosystem and this way to receive events <3